### PR TITLE
TOUCHE: Add keymapper support

### PR DESCRIPTION
**Pull request by NabeelShabbir**

### CODE REVIEW

**Comment by OMGPizzaGuy** [comment]
This looks good to me current. Let us know when this is ready from review.

File: engines/touche/metaengine.cpp [hunk 1]
```diff
@@ -104,6 +111,101 @@ void ToucheMetaEngine::removeSaveState(const char *target, int slot) const {
...
+
+		String s = ConfMan.get("language", target);
+		Language l = Common::parseLanguage(s);
+
+		act = new Action("YES", _("Press \"Yes\" Key"));
+		act->setCustomEngineActionEvent(kToucheActionYes);
+		act->addDefaultInputMapping("JOY_RIGHT_SHOULDER");
+		switch (l) {
+		case FR_FRA:
+			act->addDefaultInputMapping("o");
```
Comment (mgerhardy) [inline engines/touche/metaengine.cpp:187]: breaks are missing here

**Review by bluegr — APPROVED** [review]
Looks good, nice work! :)

**Review by OMGPizzaGuy — APPROVED** [review]

**Comment by bluegr** [comment]
Thanks!

**Pull request merged**